anise-cli/src/args.rs

```diff
@@ -1,6 +1,6 @@
 use std::path::PathBuf;
 
-use clap::{Parser, Subcommand};
+use clap::{Args as ArgsT, Parser, Subcommand};
```

Comment:
I typically prefer to rename the repo's structs to not conflict with imports, instead of the other way around. So in this case, you could rename the `pub struct Args` on line 8 to something else (`CliArgs`?), and still import clap's `Args` as is.